subcommands/keys/tuf_updates_review.go

```diff
@@ -101,20 +103,50 @@ func doTufUpdatesReview(cmd *cobra.Command, args []string) {
 				}
 			}
 		}
+	} else if updates.Status == client.TufRootUpdatesStatusNone {
+		fmt.Println("There are no TUF root updates in progress.")
+		if len(updates.Advices.Mandatory) > 0 {
+			fmt.Println(`\nFoundries.io identified some problems with your existing TUF metadata.
+Please, consider these updates in order to keep your TUF metadata functional and secure:`)
```

Comment:
"secure" is a bit too subjective. If its not secure - the TUF client/server wouldn't allow it in the first place. I'd just drop the entire sentence - cause its not something "to consider" - you *have* to do it.

Reply:
We now have no errors returned for this use case from the API (as per server-side review thread).